test/backend/DataVaultTestSuite.scala

```diff
@@ -36,10 +38,11 @@ class DataVaultTestSuite extends PlaySpec {
         "return correct response" in {
           WsTestClient.withClient { ws =>
             val uri = new URI("http://storage.googleapis.com/")
-            val vaultPath = new VaultPath(uri, HttpsDataVault.create(RemoteSourceDescriptor(uri, None), ws))
+            val vaultPath =
+              new VaultPath(uri, HttpsDataVault.create(RemoteSourceDescriptor(uri, None), ws, "example.com"))
```

Comment:
```suggestion
              new VaultPath(uri, HttpsDataVault.create(RemoteSourceDescriptor(uri, None), ws, dataStoreHost="example.com"))
```
Let’s make this a named parameter here, because I think it is not obvious from "example.com" what this is for.

Reply:
I opted for naming the variable, which I also think fulfills the purpose.